rep-0003.rst

```diff
@@ -343,6 +343,54 @@ Build System Support:
 
 - Same as Indigo
 
+Noetic Ninjemys (May 2020 - May 2025)
+--------------------------------------
+Required Support for:
+
+- Ubuntu Focal Fossa (20.04)
+
+Recommended Support for:
+
+- Debian Buster
+- Fedora 32
+
+Architectures Supported:
+
+- amd64
+- arm32
+- arm64
+
+Targeted Languages:
+
+- C++14
+- Python 3.8
+  - Testing against Python 3.7 is recommended, but not required.
+- Lisp SBCL 1.4.16
+
+Requirements:
+
++---------+------------------+----------------+-----------+
+| Package |  Ubuntu Focal   | Debian Buster  | Fedora 30 |
++=========+==================+================+===========+
+|         | Required Support |    Recommended support     |
++---------+------------------+----------------+-----------+
+| Boost   |      1.67!       |    1.67        |   1.69!   |
++---------+------------------+----------------+-----------+
+| CMake   |      3.13.4!     |    3.13.4      |   3.14.5! |
++---------+------------------+----------------+-----------+
+| Gazebo  |      11.0.0!     |    11.0.0*     |   9.5.0!  |
++---------+------------------+----------------+-----------+
+| Ogre    |      1.9!        |    1.9         |   1.9!    |
++---------+------------------+----------------+-----------+
+| OpenCV  |      3.2         |    3.2*        |   3.4.4   |
```

Comment:
Apologies if I might be misunderstanding, but I see on `packages.ubuntu.com/focal` that [libopencv-core is 4.2](https://packages.ubuntu.com/focal/libs/libopencv-core4.2) (and many other libopencv-* packages). If that means OpenCV on Focal will be 4.2, does ROS stick to 3.2.?

Reply:
Updated OpenCV row to say 4.2 from upstream for Ubuntu Focal in 6e75673